cl/settings/10-public.py

```diff
@@ -280,6 +281,14 @@
 # Max email attachment to send in bytes, 350KB
 MAX_ATTACHMENT_SIZE = 350_000
 
+# 1: Every message is BCC'ed
+# 0.5: Half of messages are BCC'ed
+# 0.1: 10% of messages are BCC'ed
+# 0: No messages are BCC'ed
+EMAIL_BCC_COPY_RATE = 0.1
+
+BCC_EMAIL_ADDRESS = "bcc@free.law"
+
```

Comment:
This is good, but it's going to be obsoleted by PR #2002 which I need to get done before this. Shouldn't be too hard to rebase and update though.